ncm-metaconfig/src/main/metaconfig/xinetd/pan/schema.pan

```diff
@@ -15,16 +17,17 @@ type xinetd_options = {
     "type" ? xinetd_options_type[]
     "flags" ? xinetd_options_flags[]
     "only_from" ? xinetd_options_ips[]
-    "cps" : long[] = list(100, 2)
+    "cps" ? long[]
```

Comment:
Removing this default has broken a number of tests that were checking for it and now probably need updating.

```
not ok 43 - Found at least one match (total 0 matches) for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$)))
not ok 44 - Order ok for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$))) (lastpos 14 before -1)
…
not ok 79 - Found at least one match (total 0 matches) for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$)))
not ok 80 - Order ok for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$))) (lastpos 13 before -1)
…
not ok 117 - Found at least one match (total 0 matches) for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$)))
not ok 118 - Order ok for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$))) (lastpos 13 before -1)
```